### DISCUSSION TITLE
Abogabot, qué tal voy?

### DISCUSSION THREAD

**Discussion by FaraN1ght** [Launch-X-Latam/MisionFrontEnd#182] [Show and tell]
Qué tal chicos, hoy avancé la práctica de Abogabot y siento que he aprendido mucho. Quisiera saber sus opiniones para retroalimentarme (mientras más técnicas sean mejor :D).

Agradezco su tiempo revisando y si quieren pueden postear sus repos también para revisarlos!

https://github.com/FaraN1ght/Abogabot/blob/main/CasoAbogabotApp.md

**Comment by doguedogue** [comment]
Se ve genial, sigue asi 🚀🚀🚀

**Comment by ghost** [comment]
se ve muy bueno, me encanta

**Comment by AleABotello** [comment]
¡Se ve muy bien!, tengo una observación en el diagrama de flujo, ya que la información de la demanda (archivo de word) y la notificación de nuevo pedido, deben de llegar al administrador y no al usuario.
